clang/unittests/Tooling/ReplacementsYamlTest.cpp

```diff
@@ -68,7 +68,7 @@ TEST(ReplacementsYamlTest, serializesNewLines) {
                "    Length:          0\n"
                "    ReplacementText: '#include <utility>\n'\n"
                "...\n",
-               YamlContentStream.str().c_str());
+               YamlContent.c_str());
```

Comment:
no difference between "then" block and "else" block?

Reply:
The Replacement text is diff.